packages/kbn-securitysolution-alerts-grouping/src/components/grouped_tables.tsx

```diff
@@ -48,14 +47,15 @@ export interface GroupingContainerProps {
     pageSize: number;
     onChangeItemsPerPage: (itemsPerPageNumber: number) => void;
     onChangePage: (pageNumber: number) => void;
+    itemsPerPageOptions: number[];
   };
   renderChildComponent: (groupFilter: Filter[]) => React.ReactNode;
   selectedGroup: string;
   takeActionItems: (groupFilters: Filter[]) => JSX.Element[];
   unit?: (n: number) => string;
 }
 
-const GroupingContainerComponent = ({
+const GroupedTablesComponent = ({
```

Comment:
Maybe we can keep `GroupingContainerComponent` or `GroupingComponent`. I want to avoid word `Table` to make the component more generic.

Reply:
i want to avoid the word `container` because we use that to title many directories in the app. We use `components` directories  for react components and `containers` directories for the logic/data/async

Reply:
I'll do `GroupingComponent`